libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.tsx

```diff
@@ -0,0 +1,11 @@
+import Highlighter from 'react-syntax-highlighter';
+
+export const SyntaxHighlighter = ({ data }: { data: unknown }) => {
+  return (
+    <div className="sh-wrapper">
```

Comment:
chore (blocking): shortened name isn't very readable, can we hvae 
```suggestion
    <div className="syntax-highlighter-wrapper">
```